cookbooks/bcpc-hadoop/recipes/journalnode.rb

```diff
@@ -6,6 +6,41 @@
 hdprel=node[:bcpc][:hadoop][:distribution][:active_release]
 hdppath="/usr/hdp/#{hdprel}"
 
+# Setup JMXTrans information
+node.default['jmxtrans'].tap do |jmxtrans|
+  jmxtrans['servers'] = [
+                          {
+                          'type': 'journalnode',
+                          'service': 'hadoop-hdfs-journalnode',
+                          'service_cmd': 'org.apache.hadoop.hdfs.qjournal.server.JournalNode'
+                          }
+                        ]
+  jmxtrans['default_queries']['journalnode'] =
+    [
+      {
+        'obj': 'Hadoop:service=JournalNode,name=RpcDetailedActivityForPort*',
```

Comment:
apparently this is not valid ruby (yet) in ruby 2.1.6p336 (the one in chefdk 0.12.0)